app/Cron_Job/Update_Option.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Boilerplate\Theme\Cron_Job;
+
+use JustCoded\WP\Framework\Objects\Cron;
+
+class Update_Option extends Cron {
```

Comment:
Please adjust according to framework code review changes.

Also, imagine some really useful case, like cleaning post revisions (like keep not more than 10) to keep database smaller. And create real example

Reply:
```suggestion
class Update_Option extends Cron {
```